doc/man/scons.xml

```diff
@@ -924,6 +924,21 @@ of a given derived file:</para>
 
 <screen>
 $ <userinput>scons --debug=includes foo.o</userinput>
+</screen>
+
+  </listitem>
+  </varlistentry>
+
+<varlistentry>
+  <term><emphasis role="bold">json</emphasis></term>
+  <listitem>
+<para>Write info to a JSON file for any of the following debug options if they are enabled: <emphasis>memory</emphasis>,
```

Comment:
Does the user need to be informed of the layout of the json file? In other words, if they want to examine the contents programmatically, do they get to deduce what will be in it, or should we tell them? (if the latter, not sure where it goes)